mmdet/structures/bbox/base_boxes.py

```diff
@@ -91,6 +91,22 @@ def convert_to(self, dst_type: Union[str, type]) -> 'BaseBoxes':
         from .box_type import convert_box_type
         return convert_box_type(self, dst_type=dst_type)
 
+    def create_empty_box(self: T,
```

Comment:
create_empty_box -> empty_boxes
also rename line in 110 from `create_fake_boxes` to `fake_boxes` to be consistent with Tensor.new_zeros().